docs/guides/index.md

```diff
@@ -22,7 +22,7 @@
   state
   wasm
   exporting
-  configuration
+  configuration/index
```

Comment:
@akshayka we should make sure there are no existing links to configuration.html otherwise they will become dead

Reply:
I did check and didn't find any